DIFD/models/difd_1year_moisture0_veg1_fence0.txt

```diff
@@ -0,0 +1,186 @@
+process_groundwater     = F                               % (F) [T/F] Enable the process of groundwater
+process_seepage_face    = F                               % (F) [T/F] Enable the process of groundwater seepage
+process_scanning        = F                               % (F) [T/F] Enable the process of scanning curves
+
+%% -------------------- [Flags Threshold] -------------------- %%
+th_grainsize            = T                               % (T) [T/F] Enable wind velocity threshold based on grainsize
+th_bedslope             = T                               % (F) [T/F] Enable wind velocity threshold based on bedslope
+th_moisture             = T                               % (F) [T/F] Enable wind velocity threshold based on moisture
+th_roughness            = F                               % (F) [T/F] Enable wind velocity threshold based on roughness
+th_nelayer              = T                               % (F) [T/F] Enable wind velocity threshold based on a non-erodible layer
+
+%% -------------------- [General physics] -------------------- %%
+g                       = 9.81                            % (9.81) [m/s^2] Gravitational constant
+v                       = 0.000015                        % (0.000015) [m^2/s] Air viscosity  
+rhoa                    = 1.225                           % (1.225) [kg/m^3] Air density
+rhog                    = 2650.0                          % (2650.) [kg/m^3] Grain density
+rhow                    = 1025.0                          % (1025.) [kg/m^3] Water density
+porosity                = 0.4                             % (0.4) [-] Sediment porosity
+cpair                   = 0.0010035                       % (0.0010035) [MJ/kg/oC] Specific heat capacity air
+
+%% -------------------- [Sediment] --------------------------- %%
+nlayers                 = 1                               % (3) [-] Number of bed layers
+layer_thickness         = 0.05000                         % (0.01) [m] Thickness of bed layers
+nfractions              = 1                               % (1) [-] Number of sediment fractions
+grain_dist              = 1.0                             % (1.) [-] Initial distribution of sediment fractions
+grain_size              = 0.00025                         % (0.000225) [m] Average grain size of each sediment fraction
+
+%% -------------------- [Wind and shear] --------------------- %%
+wind_convention         = nautical                        % (nautical ) [-] Convention used for the wind direction in the input files (cartesian or nautical)
+alfa                    = 60                              % (0)[deg] Real-world grid cell orientation wrt the North (clockwise)
+method_roughness        = constant_nikuradse              % (constant) [-] Name of method to compute roughness
+k                       = 0.03                            % (0.001) [m] Bed roughness
```

Comment:
i realized that 1 mm is too high for beaches. It should be at 6,75 mm or z0 = 0.225 mm from Hsu (1974)